plugins/outputs/opensearch/sample.conf

```diff
@@ -0,0 +1,103 @@
+# Configuration for OpenSearch to send metrics to.
+[[outputs.OpenSearch]]
+  ## URLs
+  ## The full HTTP endpoint URL for your OpenSearch instance. Multiple URLs can
+  ## be specified as part of the same cluster, but only one URLs is used to
+  ## write during each interval.
+  urls = [""]
```

Comment:
Can you please add an example for e.g. a default installation on localhost to illustrate the structure of the URL to other users!?

Reply:
Sorry @mannukalra I took the example out in a rush while re-factoring this.

Reply:
That's ok @powersj, added